File: UI/src/app/Models/message.ts
export interface Message {
  id: number;
  senderId: number;
  senderUserName: string;
  senderPhotoUrl: string;
  reciverId: number;
  reciverUserName: string;
  reciverPhotoUrl: string;
  content: string;
  dateRead: Date;
  dateSent: Date;
}

File: Dating App/API/Entities/Message.cs
namespace DatingApplication.Entities
{
    public class Message
    {
        public int Id { get; set; }
        public int SenderId { get; set; }
        public string SenderUserName { get; set; }
        public AppUser Sender { get; set; }
        public int ReciverId { get; set; }
        public string ReciverUserName { get; set; }
        public AppUser Reciver { get; set; }
        public string Content { get; set; }
        public DateTime? DateRead { get; set; }
        public DateTime DateSent { get; set; } = DateTime.UtcNow;
        public bool SenderDeleted { get; set; }
        public bool ReciverDelete { get; set; }
    }
}
